platform/pwl.h

```diff
@@ -18,9 +41,192 @@ typedef struct {
     struct wl_compositor *compositor;
 } PwlDisplay;
 
+typedef struct {
+    struct egl_display *display;
+    EGLContext context;
+    EGLConfig egl_config;
+} PwlEGLData;
```

Comment:
These are global for the Wayland connection, they should be inside `PwlDisplay`.